prExplorer/client-app/src/app/stores/productStore.ts

```diff
@@ -114,6 +117,7 @@ export default class ProductStore {
       runInAction(() => {
         this.productRegistry.delete(id);
         this.loading = false;
+        toast.success("Succesfully Deleted Product");
```

Comment:
store isn't the best place for displaying toasts. See that the store is purely related to data processing/business logic, while `toast` is something that depends on UI. 
In other words, if someone calls you store's method from a non-UI context, displaying `toast` would not be desired or may even produce some errors